### PR TITLE
feat: basic our team page

### PR DESCRIPTION
**Pull request by gregfrasco**
Created Our team page with cards and Team Sections

All Data is pulled from Strapi, not every section has people because I think we should ask them to update the Sections

image

### CODE REVIEW

File: packages/client/src/components/TeamMemberPhoto.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const colors = ['purple', 'yellow', 'blue'];
+  const selection = Math.floor(Math.random() * 3);
+  const placement = Math.floor(Math.random() * 3);
+  const firstColor = colors[selection];
+  const secondColor = colors[(selection + 1) % 3];
+  const thirdColor = colors[(selection + 2) % 3];
+
+  return (
+    <Box>
+      <Box
```
Comment (vpsx) [inline packages/client/src/components/TeamMemberPhoto.jsx]: Are there supposed to be two boxes?

File: packages/client/src/components/TeamMemberPhoto.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { Avatar, Box } from '@mui/material';
+import { prependStrapiURL } from '../utils.jsx';
+
+export const TeamMemberPhoto = ({ Name, Photo, index }) => {
+  const colors = ['purple', 'yellow', 'blue'];
+  const selection = Math.floor(Math.random() * 3);
+  const placement = Math.floor(Math.random() * 3);
+  const firstColor = colors[selection];
+  const secondColor = colors[(selection + 1) % 3];
+  const thirdColor = colors[(selection + 2) % 3];
```
Comment (vpsx) [inline packages/client/src/components/TeamMemberPhoto.jsx]: Was this idea dropped or is this here for future use? Also, :star_struck: :star_struck: haha

File: packages/client/src/components/Team.jsx [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
+  return (
+    <Stack spacing={10}>
+      <Box>
+        <Typography variant="h3" component="h1" gutterBottom>
+          Meet Our Team
+        </Typography>
+        <Typography>
+          At our hospital, we have assembled a team of highly skilled and compassionate professionals who are dedicated
+          to providing exceptional psychiatric services. With years of experience and a deep understanding of mental
+          health, our team is committed to helping patients on their journey to wellness.
```
Comment (vpsx) [inline packages/client/src/components/Team.jsx:22]: Just flagging that this is not something they wrote - hopefully nobody will be alarmed

File: packages/client/src/components/Layout.jsx [hunk 4]
```diff
@@ -1,13 +1,10 @@
-import { useState } from 'react'
```
Comment (vpsx) [inline packages/client/src/components/Layout.jsx:1]: ooooops thank you

**Review by vpsx — APPROVED** [review]
Thanks Greg!!!! Left some questions/notes but LGTM! I would just have split out the linting (and maybe the footer) into separate commits :smiling_face_with_tear:

File: packages/client/src/components/TeamMemberPhoto.jsx (hunk 2 above)
Comment (gregfrasco) [reply packages/client/src/components/TeamMemberPhoto.jsx]: Oh I tried to add some color, in here but it didn't work. I will remove for now.

File: packages/client/src/components/Team.jsx (hunk 3 above)
Comment (gregfrasco) [reply packages/client/src/components/Team.jsx:22]: We can flag it and either change it or remove it all together

File: packages/client/src/components/TeamMemberPhoto.jsx (hunk 1 above)
Comment (gregfrasco) [reply packages/client/src/components/TeamMemberPhoto.jsx]: Nope, fixed

**Pull request merged**